src/components/CreateNote/CreateNoteContainer.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
```

Comment:
please import only needed elements from react
```suggestion
import { useState } from 'react';
```